lib/s3-sync.js

```diff
@@ -123,7 +126,6 @@ module.exports = class S3Sync extends EventEmitter {
   }
 
   announce(msg) {
-    this.status = msg
```

Comment:
https://github.com/superhuman/zen/blob/9a2a67be3e774c1f362cb34a103faf6d995a3cc5/lib/server.js#L174 status is used here. I think removing it could cause an issue. Maybe we nust remove the log.

Reply:
oh good find!